autogen.sh

```diff
@@ -1033,7 +1032,7 @@ if [ "$do_build_configure" = "yes" ] ; then
                 else
                     echo "failed"
                 fi
-                echo_n "Patching libtool.m4 for compatibility macOS Big Sur..."
+                echo_n "Patching libtool.m4 for compatibility macOS BigSur..."
```

Comment:
```suggestion
                echo_n "Patching libtool.m4 for compatibility macOS Big Sur..."
```

Why the change here?

Reply:
"Sur" triggers spell checker.

Reply:
Then the spell checker is wrong. 😄 The name of the OS is "macOS Big Sur".

Surely there's a way to add terms to the dictionary?

Reply:
Yes. I am looking at the balance of "local ugliness of a hack" vs. "the long-range maintenance impact of a *correct* solution" (and who is caring for it).